### DISCUSSION TITLE
安卓新版本能否上传至Google Play

### DISCUSSION THREAD

**Discussion by SolidBlock-cn** [fcitx5-android/fcitx5-android#548] [Q&A | 问答] [answered]
我是从Google Play上下载的本软件，但Google Play上还没有更新到0.0.9，从GitHub Release和F-droid中下载的版本，均会导致签名不一致。

**Comment by rocka** [comment, marked as answer]
**TL;DR:** 不能。

----

Google Play 的新政策要求开发者填写真实姓名和地址，并公开展示在商店页面： https://support.google.com/googleplay/android-developer/answer/14177239#verification_reqs_personal

因为没有人愿意写自己的个人信息，所以 Google Play 版本以后不会再更新了。请在高级设置里选择“导出用户数据“，并迁移到 GitHub Release 或 F-Droid 的版本。这两个版本签名一致，可以互相覆盖安装。
